clusters/aat/base/kustomization.yaml

```diff
@@ -21,6 +21,7 @@ resources:
   - ../../../apps/hmc/base/kustomize.yaml
   - ../../../apps/probate/base/kustomize.yaml
   - ../../../apps/em/base/kustomize.yaml
+  - ../../../apps/et/base/kustomize.yaml
```

Comment:
you've modified some perftest files but you haven't added this line to perftest? bit confusing, especially with the title being add it to AAT

Reply:
This is in the perftest already https://github.com/hmcts/cnp-flux-config/blob/ae11f48974426bdc5adbc2942727a922cd686ff6/clusters/perftest/base/kustomization.yaml#L15

The ask was for me to help sort AAT out, but while doing that I have had errors on the CI from demo and perftest, so correcting them at the same time.